incubator/external-dns/templates/deployment.yaml

```diff
@@ -0,0 +1,70 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ template "fullname" . }}
+  labels: {{ include "labels" . | indent 4 }}
+spec:
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels: {{ include "labels" . | indent 8 }}
```

Comment:
this will include the chart: label that will include the chart version.  This will make upgrades fail.